.evergreen/scripts/generate-parallel-e2e-tasks.js

```diff
@@ -150,7 +151,9 @@ const main = () => {
   // If the task is triggered by a upstream task, we should always generate the tasks
   const shouldAlwaysGenerateTasks = ALWAYS_GENERATE_TASKS_REQUESTERS.includes(requester);
   const isDownstreamTask = triggerId !== undefined && triggerId !== "";
```

Comment:
nit. I don't think you need `TRIGGER_ID` anymore since it seems it's logically equivalent to `trigger` requester (a case you already covered)